front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/AboutReportWidget/index.tsx

```diff
@@ -13,47 +13,33 @@ import Text from 'components/admin/ContentBuilder/Widgets/Text';
 import TenantLogo from 'containers/MobileNavbar/TenantLogo';
 import { NoWidgetSettings } from 'components/admin/ContentBuilder/Widgets/NoWidgetSettings';
 
-// utils
-import { useIntl } from 'utils/cl-intl';
-import { isNilOrError } from 'utils/helperUtils';
-import moment from 'moment';
-
 // hooks
 import useReport from 'hooks/useReport';
 import useUser from 'hooks/useUser';
 import useProject from 'hooks/useProject';
-import usePhases from 'hooks/usePhases';
```

Comment:
Have we removed phases because the dates are now being set from the templates?

Reply:
Yeah, this way they are only set once (when the report is created), so if you make any changes to the phases later they're not reflected in these date periods. That's also how I would expect the date filters to behave